code/modules/mob/living/simple_animal/guardian/types/standard.dm

```diff
@@ -2,6 +2,7 @@
 /mob/living/simple_animal/hostile/guardian/punch
 	melee_damage_lower = 20
 	melee_damage_upper = 20
+	next_move_modifier = 0.9 //attacks 20% faster
```

Comment:
from 100% (1) to 90% (0.9) of attack delays is a decrease in time by 10%, not 20%